pkg/controller/logstorage/linseed/linseed_controller.go

```diff
@@ -173,10 +173,13 @@ func Add(mgr manager.Manager, opts options.AddOptions) error {
 
 	k8sClient, err := kubernetes.NewForConfig(mgr.GetConfig())
 	if err != nil {
-		return fmt.Errorf("log-storage-elastic-controller failed to establish a connection to k8s: %w", err)
+		return fmt.Errorf("log-storage-linseed-controller failed to establish a connection to k8s: %w", err)
 	}
 
 	go utils.WaitToAddTierWatch(networkpolicy.TigeraComponentTierName, c, k8sClient, log, r.tierWatchReady)
+	go utils.WaitToAddNetworkPolicyWatches(c, k8sClient, log, []types.NamespacedName{
+		{Name: linseed.PolicyName, Namespace: render.ElasticsearchNamespace},
```

Comment:
For multi-tenant clusters, `render.ElasticsearchNamespace` is wrong here - it should probably use this: https://github.com/tigera/operator/blob/ddeed95e5f6e1f1f3574c2b46def5c7660c1257d/pkg/controller/policyrecommendation/policyrecommendation_controller.go#L86

Reply:
There are a few types of watches:

- Watches that are always in the Install namespace (helper.InstallNamespace())
- Watches that are always in the truth namespace (helper.TruthNamespace())
- Watches that are either in a specific namespace for single-tenant, and _all_ namespaces for multi-tenant (like this one)